lib/spider/spider/in_context_learning.py

```diff
@@ -0,0 +1,128 @@
+import json
```

Comment:
Can you add a docstring where you show a command on how to start this script with hydra or explain which configs are used, etc. please? 🙏